test/common/performance/common/executor.go

```diff
@@ -16,8 +16,27 @@ limitations under the License.
 
 package common
 
-import "context"
+import (
+	"context"
+	"sync"
+)
 
 type Executor interface {
 	Run(ctx context.Context)
 }
+
+type Executors []Executor
+
+func (e Executors) Run(ctx context.Context) {
```

Comment:
Have you seen https://godoc.org/golang.org/x/sync/errgroup? Similar but adds error handling.

Reply:
Honestly I wasn't aware of, I prefer to stick with this solution till we need some more accurate error handling